docs/user/library/data/geopackage.rst

```diff
@@ -25,31 +25,54 @@ References:
     </dependency>
 
 
-DataStore Connection Parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+DataStore API
+^^^^^^^^^^^^^
 
-============== ============================================
-Parameter      Description
-============== ============================================
-``dbtype``     Must be the string ``geopkg``
-``database``   The database to connect to
-``user``       User name (optional)
-============== ============================================
+GeoTools provides direct access to JDBCDataStore implementation for accessing feature contents as expressed in the specification:
 
-Access
-^^^^^^
+> A GeoPackage with a ``gpkg_contents`` table row with a "features" data_type SHALL contain a ``gpkg_geometry_columns`` table per Table 5 and ``gpkg_geometry_columns`` Table Definition SQL.
 
-Example use::
+Spatial index is supported with the use of ``gpkg_rtree_index``.
+
+The ``JDBCDataStore.createVirtualTable`` functionality is not supported (as the ``gpkg_geometry_columns`` information is not available for ad-hoc SQL queries).
+
+Connection Parameters
+'''''''''''''''''''''
+
+.. list-table:: Connection Parameters
+   :widths: 30 79
+   :header-rows: 1
+
+   * - Parameter
+     - Quantity
+   * - ``dbtype``
+     - Must be the string ``geopkg``
+   * - ``database``
+     - The database to connect to
```

Comment:
I expect it is the path also.

Reply:
Confirm, it's also the path. If a base directory is provided, then the db name is searched inside that directory (does not seem that useful to me though, but it's what the factory initializer in the gs-gpkg module does).

Reply:
Yeah we really should move that factory initializer into geoserver core then.

Reply:
I really don't see the value of it. Would wipe it out instead. Do you have anyone that puts geopackages in the data directory? Storing data in the DD is fine to send around small projects, but on production with large datasets (or with many of them) it's a liability.

Reply:
I swapped my default training databases to include geopackage rather than shapefile in the data directory; also mount  `GEOSERVER_DATA_DIR/data` folder out for WEBDAV access, since it is used by default for some of the REST API.

Reply:
Our training packages explicitly removed any trace of data from the data dirs, as people learning that way start with the wrong habit and will come to regret it once in production. But I guess, to each his own.

Reply:
Yeah I wanted to see if we could list short cuts to other locations in the file chooser.